libr/asm/d/arm.sdb.txt

```diff
@@ -1,167 +1,475 @@
-tst=test
-ubfx=unsigned bit field extract
-uxtab16=zero extend bytes to halfwords add halfwords
-uxtab=zero extend byte to word, add
-vadd=vector add
-vsri=vector shift right and insert
-wfs=armfpu: write fp status register. wfs r0
-cpsie=like msr, but only enable interrupts (i or f)
-cpsid=like msr, but only disable interrupts (i or f)
-uxth=extracts bits[15:0] and zero extends to 32 bits.
-sxth=extracts bits[15:0] and sign extends to 32 bits.
-uxtb=extracts bits[7:0] and zero extends to 32 bits.
-sxtb=extracts bits[7:0] and sign extends to 32 bits.
-sxtw=signed extend word
-pop=load N elements from stack into given registers
-vldmia=vector load memory into paired registers
-pop.w=load N words from stack into registers
-it=if then
-bic=logical bit clear
-bics=Bitwise bit clear (shifted register), setting the condition flags.
-bics.w=word bitwise bit clear and update flags
-udiv=unsigned division
-sdiv=signed division
-mul=multiply
-mla=multiply and accumulate
-mls=multiply and substract
-rsbs=subtracts the value from zero, producing the arithmetic negative of the value, and updates flags
-rsbs.w=substract the negative value
-uxtab=zero extend Byte and Add. Extends an 8-bit value to a 32-bit value.
-uxtb.w=zero textend word and add, extend to 32bit value
-wfi=wait for interrupt
-wfe=wait for event
-yield=alias for hint instructrion
-dc=data cache operation (alias of sys)
-ic=instruction cache operation (alias of sys)
-dsb=data synchronization barrier.
-isb=instruction synchronization barrier.
-movz=move shifted 16-bit immediate to register.
-hlt=external debug breakpoint.
-hvc=hypervisor call
-eret=return from exception
-paciasp=function prelude for pointer authentication using A key
-pacibsp=function prelude for pointer authentication using B key
-autiasp=authenticate return pointer before returning from function using A key
-autibsp=authenticate return pointer before returning from function using B key
-xpaci=strip pac from instruction pointer
-xpacd=strip pac from data pointer
-retaa=return with authenticated pointer using A key
-retab=return with authenticated pointer using B key
-xpaclri=system strip pointer authentication
-blraa=brank and link with pointer authentication
-blraaz=variants of blraa
-blrab=variants of blraa
-blrabz=variants of blraa
-braa=branch with pointer authentication using A key
-braaz=variant of braa
-brab=branch with pointer authentication using B key
-brabz=variant of brabz
-stadd=atomic add word (arm v8.1)
-staddl=atomic add dword (arm v8.1)
-cbnz=compare and branch on non-zero
-cbz=compare and branch on zero
-adcs=add with carry and update flags
-ldm=load to multiple registers from memory
-stm=store multiple registers to memory
-stp=store register pairs in memory
-ldp=load a pair of registers from memory
-addlo=add if C is clear (no carry)
-mvn=move bitwise NOT
-mvns=move bitwise NOT and update flags
-blo=variant of bcc
-bhs=variant of bcs
+adc=Add with Carry
+adcs=Add with Carry, setting flags
+add=Add
```

Comment:
not very descriptive. old text was clearer in this case
![Screenshot 2021-07-02 at 11 50 46](https://user-images.githubusercontent.com/917142/124256607-ca5b7480-db2b-11eb-8e76-1c8f5d84f47d.png)